modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/DynamicInlineScroll.es.js

```diff
@@ -75,11 +76,14 @@ class DynamicInlineScroll extends PortletBase {
 	 * @protected
 	 */
 	addListItem_(listElement, pageIndex) {
-		const listItem = document.createElement('li');
+		let listItem = listElement.querySelector('li');
+		listItem = listItem.cloneNode(true);
 
-		listItem.append(
-			`<a href="${this.getHREF_(pageIndex)}">${pageIndex}</a>`
-		);
+		const listItemLink = listItem.querySelector('a');
+		listItemLink.setAttribute('href', this.getHREF_(pageIndex));
+		listItemLink.textContent = pageIndex;
```

Comment:
I believe we have an SF rule that should be enforced to leave an empty line between variable declaration and usage.

Reply:
Updated